### PR TITLE
Add configurable timeout as a gateway account setting

### PR DESCRIPTION
**Pull request by XaosSintez**

### CODE REVIEW

File: spec/components/schemas/GatewayAccount.yaml [hunk 1]
```diff
@@ -105,6 +105,10 @@ properties:
     description: Set restriction conditions on transaction
     allOf:
       - $ref: "#/components/schemas/Condition"
+  timeout:
+    description: Gateway Account request timeout
+    type: integer
```
Comment (ashkarpetin) [inline spec/components/schemas/GatewayAccount.yaml]: Maybe add min and mix limit?

File: spec/components/schemas/GatewayAccount.yaml [hunk 2]
```diff
@@ -105,6 +105,12 @@ properties:
     description: Set restriction conditions on transaction
     allOf:
       - $ref: "#/components/schemas/Condition"
+  timeout:
+    description: Gateway Account request timeout
```
Comment (colinbird) [inline spec/components/schemas/GatewayAccount.yaml]: seconds, milliseconds?

Comment (colinbird) [reply spec/components/schemas/GatewayAccount.yaml]: i guess now you can tell from the max, but would still be good to add "in seconds"

Comment (ashkarpetin) [inline spec/components/schemas/GatewayAccount.yaml]:
```suggestion
    description: Gateway Account request timeout in seconds
```

**Pull request merged**